src/libs/WorkflowUtils.ts

```diff
@@ -191,22 +196,44 @@ function convertApprovalWorkflowToPolicyEmployees({approvalWorkflow, membersToRe
 
     approvalWorkflow.approvers.forEach((approver, index) => {
```

Comment:
It would be really nice at some point to get a bunch of detailed comments to explain the intention of all the logic in this method. It would be difficult for someone to maintain this cold if they didn't have any context behind the feature. Maybe you could do that in a future cleanup PR?

Reply:
I agree! Let's do it in the cleanup PR where the beta will be removed and some logic cleaned 👍